diff -Naur apprise-0.9.6/test/helpers/rest.py apprise-0.9.6.patched/test/helpers/rest.py
--- apprise-0.9.6/test/helpers/rest.py	2021-12-01 20:00:27.000000000 -0500
+++ apprise-0.9.6.patched/test/helpers/rest.py	2021-12-02 15:05:54.957547670 -0500
@@ -54,8 +54,6 @@
             0, 'requests.RequestException() not handled'),
         requests.HTTPError(
             0, 'requests.HTTPError() not handled'),
-        requests.ReadTimeout(
-            0, 'requests.ReadTimeout() not handled'),
         requests.TooManyRedirects(
             0, 'requests.TooManyRedirects() not handled'),
     )
diff -Naur apprise-0.9.6/test/test_attach_http.py apprise-0.9.6.patched/test/test_attach_http.py
--- apprise-0.9.6/test/test_attach_http.py	2021-12-01 20:00:27.000000000 -0500
+++ apprise-0.9.6.patched/test/test_attach_http.py	2021-12-02 15:05:48.165575030 -0500
@@ -51,8 +51,6 @@
         0, 'requests.RequestException() not handled'),
     requests.HTTPError(
         0, 'requests.HTTPError() not handled'),
-    requests.ReadTimeout(
-        0, 'requests.ReadTimeout() not handled'),
     requests.TooManyRedirects(
         0, 'requests.TooManyRedirects() not handled'),
 
diff -Naur apprise-0.9.6/test/test_config_http.py apprise-0.9.6.patched/test/test_config_http.py
--- apprise-0.9.6/test/test_config_http.py	2021-12-01 20:00:27.000000000 -0500
+++ apprise-0.9.6.patched/test/test_config_http.py	2021-12-02 15:06:01.927519591 -0500
@@ -46,8 +46,6 @@
         0, 'requests.RequestException() not handled'),
     requests.HTTPError(
         0, 'requests.HTTPError() not handled'),
-    requests.ReadTimeout(
-        0, 'requests.ReadTimeout() not handled'),
     requests.TooManyRedirects(
         0, 'requests.TooManyRedirects() not handled'),
 )
diff -Naur apprise-0.9.6/test/test_plugin_glib.py apprise-0.9.6.patched/test/test_plugin_glib.py
--- apprise-0.9.6/test/test_plugin_glib.py	2021-12-01 20:00:27.000000000 -0500
+++ apprise-0.9.6.patched/test/test_plugin_glib.py	2021-12-02 15:06:40.823362895 -0500
@@ -48,7 +48,7 @@
 
 if 'dbus' not in sys.modules:
     # Environment doesn't allow for dbus
-    pytest.skip("Skipping dbus-python based tests", allow_module_level=True)
+    pytest.skip("Skipping dbus-python based tests")
 
 from dbus import DBusException  # noqa E402
 
