diff -Naur apprise-1.6.0/test/test_plugin_macosx.py apprise-1.6.0.patched/test/test_plugin_macosx.py
--- apprise-1.6.0/test/test_plugin_macosx.py	2023-12-22 16:51:24.000000000 -0500
+++ apprise-1.6.0.patched/test/test_plugin_macosx.py	2023-12-22 17:38:35.720131819 -0500
@@ -42,9 +42,8 @@
 logging.disable(logging.CRITICAL)
 
 
-if sys.platform not in ["darwin", "linux"]:
-    pytest.skip("Only makes sense on macOS, but testable in Linux",
-                allow_module_level=True)
+if sys.platform != "darwin":
+    pytest.skip("MacOS test only", allow_module_level=True)
 
 
 @pytest.fixture
